Listed Hashicorp Plugin Page

DARK-art108 opened the pull request:
# What does this change
Added Docs for Integrating Porter with Hashicorp Vault.

# What issue does it fix
Closes #1587 

[1]: /CONTRIBUTING.md#when-to-open-a-pull-request

# Notes for the reviewer
_Put any questions or notes for the reviewer here._

# Checklist
- [ ] Unit Tests
- [x] Documentation
- [ ] Schema (porter.yaml)

If this is your first pull request, please add your name to the bottom of our [Contributors][contributors] list. Thank you for making Porter better! 🙇‍♀️

[contributors]: /CONTRIBUTORS.md

DARK-art108 commented:
@carolynvs Added Hashicorp Plugin Page Take a look at it! :)

carolynvs commented on docs/content/plugins/hashicorp.md:
```diff
@@ -0,0 +1,40 @@
+description: Integrate Porter with Hashicorp
+---
+
+<img src="/images/mixins/hashicorp.png" class="mixin-logo" style="width: 300px"/>
```
The image should go in `/images/plugins` and this link updated with the new location.

carolynvs reviewed (commented):
Looks good, just one small change and this is ready to merge.

DARK-art108 commented:
@carolynvs Added suggested Changes, there is no Plugins folder previously, I have added it and dropped the png into that and updated the path! Take a look at it :)

carolynvs reviewed (approved):
LGTM, thanks!
